ETC.tsv

```diff
@@ -12473,9 +12473,9 @@ ETC_20150714_012474	Green Demonic Power Crystal
 ETC_20150714_012475	Setting the pots of flame
 ETC_20150714_012476	There are already jewels of prominence nearby
 ETC_20150714_012477	You've discovered the jewel of prominence!
-ETC_20150714_012478	You've defeated the disturbing Beakaras!{nl}Find usable parts from the preliminary purification device
+ETC_20150714_012478	You've defeated the Beakaras!{nl}Get the parts from the spare purifier nearby.
```

Comment:
You've defeated the Beakaras! -> You have defeated Bearkaras!

The original does say find for/search, though.